src/simulation.md

```diff
@@ -307,27 +306,34 @@ Analogous to the `door_supervisor`, a `lift_supervisor` [node](https://github.co
 
 
 #### Workcells
-Robots performing deliveries within facilities is a common use case and hence a `Delivery` task is configured into the `rmf_fleet_adapters`. In a delivery task, a payload is loaded onto the robot at one location and unloaded at another. The loading and unloading of the payload may be automated by a robot/workcell in a facility. To integrate these systems (generalized as "dispensers") with RMF core systems, a set of dispenser messages are [defined](https://github.com/osrf/rmf_core/tree/master/rmf_dispenser_msgs/msg). When the robot reaches the loading station, the `rmf_fleet_adapter` publishes a `DispenserRequest` message which the dispenser receives and begins processing. When the dispensing is successful, the dispenser is required to publish a `DispenserResult` message with `SUCCESS` status. The `rmf_fleet_adapter` then instructs the robot to proceed to the unloading station where a similar set of messages are exchanged with the workcell unloading the payload.
+Robots performing deliveries within facilities is a common use case and hence a `Delivery` task is configured into the `rmf_fleet_adapters`. In a delivery task, a payload is loaded onto the robot at one location and unloaded at another. The loading and unloading of the payload may be automated by a robot/workcell in the facility. To integrate these systems with RMF core systems, a set of dispenser messages are [defined](https://github.com/osrf/rmf_core/tree/master/rmf_dispenser_msgs/msg). Despite their names, the messages are generalized for use by systems other than dispensers as well. When a robot reaches the loading station, its `rmf_fleet_adapter` publishes a `DispenserRequest` message which the station receives and begins processing. When the loading is successful, the station publishes a `DispenserResult` message with `SUCCESS` status. The `rmf_fleet_adapter` then instructs the robot to proceed to the unloading station where a similar set of messages are exchanged with the workcell unloading the payload.
```

Comment:
```suggestion
Robots performing deliveries within facilities is a common use case and hence a `Delivery` task is configured into the `rmf_fleet_adapters`. In a delivery task, a payload is loaded onto the robot at one location and unloaded at another. The loading and unloading of the payload may be automated by a robot/workcell in the facility. To integrate these systems with RMF core systems, a set of dispenser messages are [defined](https://github.com/osrf/rmf_core/tree/master/rmf_dispenser_msgs/msg). Despite their names, the messages are generalized for use by systems other than dispensers as well. When a robot reaches the loading station, its `rmf_fleet_adapter` publishes a `DispenserRequest` message which the station receives and begins processing. When the loading is successful, the station publishes a `DispenserResult` message with `SUCCESS` status. The `rmf_fleet_adapter` then instructs the robot to proceed to the unloading station where a similar set of messages are exchanged with the workcell unloading the payload.
```

Reply:
Are `station` and `waypoint` interchangeable here? Or is one a location and one is a function? I'm getting a little mixed up on terminology

Reply:
Sorry I totally mixed up my terminology here. There are pickup and dropoff waypoints with loading and unloading workcell at each respectively. I have removed instances of `station` and updated the para https://github.com/osrf/ros2multirobotbook/pull/25/commits/1b04be24cd34e4eefc5c74855877204f855f8acd